.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "stkb.rewrap"
+    ]
+}
```

Comment:
EOF newline missing!

Please add relevant Github workflows to ensure these issues are caught automatically

https://github.com/ai-cfia/github-workflows/blob/main/.github/workflows/workflow-repo-standards-validation.yml